### DISCUSSION TITLE
Redis string values

### DISCUSSION THREAD

**Discussion by runonma** [kedacore/keda#1225] [Ideas]
Hi, have you considered adding Redis string type to the Redis scaler? Sometimes a relevant metric might be stored as a flat int in a string type key.

**Comment by zroubalik** [comment]
Hi, I don't see a problem in this. Are you willing to contribute this one? And feel free to open an issue with the proposal.

**Comment by avneeshkumar** [comment]
Any update on this ? I am also looking for something similar ?
